### PR TITLE
Backend Tensorflow 1.x: replace tf.layers.dense to tf.keras.layers.Dense

### PR DESCRIPTION
**Pull request by vl-dud**
Switch to the modern TF style, which avoids such warnings: 
```
UserWarning: `tf.layers.dense` is deprecated and will be removed in a future version. Please use `tf.keras.layers.Dense` instead.
```

### CODE REVIEW

**Comment by lululxvi** [comment]
Please see below for the regularizaion:
https://github.com/lululxvi/deepxde/blob/ba8e8240ed7ab4ed9d30316409a9f20713027bbc/deepxde/nn/tensorflow_compat_v1/nn.py#L14

https://github.com/lululxvi/deepxde/blob/ba8e8240ed7ab4ed9d30316409a9f20713027bbc/deepxde/nn/tensorflow_compat_v1/fnn.py#L118

**Comment by vl-dud** [comment]
You are right, the loss of regularization was ignored. I have corrected it.
